Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -435,6 +449,154 @@ void EditorActionsHandler::InitializeActions()
         );
     }
 
+    // --- Edit Actions
+
+    // Undo
+    {
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "&Undo";
+        actionProperties.m_description = "Undo last operation";
+        actionProperties.m_category = "Edit";
+        actionProperties.m_hideFromMenusWhenDisabled = false;
+
+        m_actionManagerInterface->RegisterAction(
+            EditorMainWindowActionContextIdentifier,
+            "o3de.action.edit.undo",
```

Comment:
Q: `"o3de.action.edit.undo"` can this not be a named constant like the others above? Same for the others below.

Reply:
It can, but there's so many :'D